controller/assignment.js

```diff
@@ -0,0 +1,53 @@
+import {
+  createAssignment, deleteAssignmentById, assignmentList, updateAssignmentById,
+} from "#services/assignment ";
```

Comment:
just one tiny change, this is causing test cases to fail, please remove this space. and use setup.js, otherwise random errors like this would happen.
```suggestion
} from "#services/assignment";
```